markdownlint/TOP005_blanksAroundMultilineHtmlTags/TOP005_blanksAroundMultilineHtmlTags.js

```diff
@@ -63,15 +63,20 @@ module.exports = {
         replacementText = `${replacementText}\n`;
       }
 
+      const ERROR_MESSAGE_BASE = "Expected a newline (\\n) or a code block delimiter (```)";
+
+      const detail = ERROR_MESSAGE_BASE
+        .concat(lineBeforeIsValid ? "" : " before the tag")
+        .concat(lineBeforeIsValid || lineAfterIsValid ? "" : " and")
+        .concat(lineAfterIsValid ? "" : " after the tag");
+      
```

Comment:
Nit - given that this bit follows the same general logic as the bit above in terms of conditionally altering the replacementText, for constistency, it should be written in the same sort of format.

i.e. instead of having two very similar bits of logic handled via completely different means, they ought to be handled the same way, in this case, `let` variable and conditionals with template strings, rather than suddenly using concat methods and ternaries.

Reply:
I personally prefer no-mutation when I can, but I agree with you that consistency is definitely important! I have refactored it

Reply:
The specific approach actually used doesn't matter so much so long as it's within reason. More importantly it's consistency else anyone else working on this file can get more confused when two things that do essentially the exact same thing are written in two completely different ways.

You might want to test the latest code though :P The `const` looks suspicious.

Reply:
Oh yea oopsie! I fixed it!